src/state_history_pg.hpp

```diff
@@ -39,7 +39,7 @@ inline eosio::checksum256 sql_to_checksum256(const char* ch) {
     std::vector<uint8_t> v;
     boost::algorithm::unhex(ch, ch + strlen(ch), std::back_inserter(v));
     eosio::checksum256 result;
-    if (v.size() != result.value.size())
+    if (v.size() != sizeof(result.value))
         throw std::runtime_error("hex string has incorrect length");
     memcpy(result.value.data(), v.data(), result.value.size());
```

Comment:
Here it should be `sizeof(result.value)` instead of `result.value.size()` too.

Reply:
that right, but it work now. so i think the sha256 not use in other wise